src/classes/STG_SettingsManager_CTRL.cls

```diff
@@ -115,7 +120,7 @@ public with sharing class STG_SettingsManager_CTRL extends STG_Panel {
         if (String.isBlank(STG_Panel.stgService.stgAllo.Default__c)) {
             return;
         }
-
+       
```

Comment:
Looks like there's some extra white space here. You can configure VSCode to automatically fix these: https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut